spec/rubocop/cop/rails/duplicate_association_spec.rb

```diff
@@ -16,8 +16,8 @@ class Post < ApplicationRecord
 
       expect_correction(<<~RUBY)
         class Post < ApplicationRecord
-          belongs_to :bar
           belongs_to :foo
+          belongs_to :bar
```

Comment:
Although this change caught my attention, the later definition takes precedence, so let's proceed with this for now.